lib/rspec/abq/extensions.rb

```diff
@@ -125,7 +125,12 @@ def run_specs(example_groups)
             end
           end
 
-          exit_code(examples_passed)
+          if Abq.target_test_case == Abq.end_marker
```

Comment:
oh this is a bug. whoops

Reply:
```suggestion
          if Abq.target_test_case == Abq::TestCase.end_marker
```

Reply:
I'm not sure how this didn't break the build though ...

Reply:
that's a bug in ABQ